### ISSUE TITLE
dictify method drops custom columns added in includes param

### ISSUE THREAD

**Issue by grant-humphries** [stefanofontanelli/ColanderAlchemy#98]
This example in from docs:

```py
import deform
import colander
from colanderalchemy import SQLAlchemySchemaNode
from my.project import SomeClass

typ = colander.String()
widget = deform.widget.SelectWidget(values=(('foo', 'a'),
                                            ('bar', 'b'),
                                            ('baz', 'c')))
column = colander.SchemaNode(typ,
                             name='customfield',
                             widget=widget)
schema = SQLAlchemySchemaNode(SomeClass,
                              includes=['name', column, 'biography'],
                              excludes=['id'],
                              title='Some class')
```

Illustrates that you can add custom columns that do not exist on the provided SQLAlchemy table to the resultant colander schema by passing it in the includes list.  However the `dictify` method uses an inspector object based on that SQLA table to verify attributes, thus any columns added only in the includes will be dropped by the method.  See the [key portion](https://github.com/stefanofontanelli/ColanderAlchemy/blob/master/colanderalchemy/schema.py#L595) of the code here:

```py
try:
    getattr(self.inspector.column_attrs, name)
    value = getattr(obj, name)

except AttributeError:
    try:
        prop = getattr(self.inspector.relationships, name)
        if prop.uselist:
            value = [self[name].children[0].dictify(o)
                     for o in getattr(obj, name)]
        else:
            o = getattr(obj, name)
            value = None if o is None else self[name].dictify(o)
```

**Comment by grant-humphries** [comment]
Thinking about this more maybe it doesn't make sense for `dictify` to take into account custom columns as their values can could be added to the resultant object.  Am curious to get other folks' opinions on this.

**Comment by tisdall** [comment]
`dictify` was intended to take an ORM object and return it as a `dict`.  I'm not really sure it makes sense to look for attributes that aren't part of the model.  Where would the value come from if it's not in the model?  ... maybe I'm just not understanding your use-case.

**Comment by grant-humphries** [comment]
I was writing a sqlalchemy query with a table that I've used to with `SQLAlchemySchemaNode` to create a colander schema.  In that query I return a couple of columns from other tables and tacked those onto the sqlalchemy object instance as attributes.  

I then manually created schema nodes for those additional fields and passed them in the `includes` when I created the sqla schema node and then tried to dictify the object on which I had added those attributes.

I realize now that this is probably not a good approach and I can add the extra query attributes to the resultant dictionary just as easily and then serialize that `dict` object.  I'll go ahead and close this issue
